common/src/main/scala/explore/common/TargetListGroupQueries.scala

```diff
@@ -38,7 +39,7 @@ object TargetListGroupQueries {
   // This is used for sorting the TargetListGroupObsList. If we change to sort by name or something
   // else, we can remove this.
   implicit val orderSortedSet: Order[TargetEnvIdObsIdSet] =
```

Comment:
Although I came up with the name `TargetEnvIdObsId`, I'm not too happy with it. If anyone can think of a better one  it would be welcome.

Reply:
Well, it is a `TargetEnv.id`. So we could call it a `TargetEnvId`. I think on the UI we just need to be aware that a `TargetEnv` is not a `TargetEnvironment`. We already have that distinction and I don't think it is an issue in practice.

Reply:
Except we're already using `TargetEnvIdSet` for a `NonEmptySet[TargetEnvironment]`

Reply:
Will do in separate PR if we do it.